src/App.css

```diff
@@ -0,0 +1,11 @@
+body {
+  margin: 0;
+  padding: 0;
+  background-color: #fafafa;
+  font-family: Montserrat Arial, sans-serif;
+  font-stretch: normal;
```

Comment:
- [ ] Kindly please import these two font-family (**`RobotoSlab, Montserrat`** ) to your CSS file that you used in your app, I couldn't see the effect because they are not part of css default font-family. 

A typical example using googlefont 👉 `@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300;400;500;600;700&family=Roboto+Slab:wght@100;200;300;500;600;700&display=swap");`